assets/css/_vehicle_map.scss

```diff
@@ -79,12 +79,27 @@
 .m-vehicle-map__stop {
   fill: $color-stop-fill;
   fill-opacity: 1;
+
   stroke: $color-stop-stroke;
-  stroke-width: 1;
-}
+  stroke-width: 2px;
+
+  &:hover {
+    stroke-width: 4px;
+  }
 
-.m-vehicle-map__stop:focus {
-  outline: 0;
+  &:focus {
+    stroke-width: 4px;
+    box-shadow: 0px 0px 0px 1px #514f55;
```

Comment:
Figma designs used this box-shadow property, but it doesn't seem to actually be applying for me, so I added the filter below, but I've got to run that by design first.

Ideally design can help me decide what to do, but this line should likely be removed.

Reply:
[`6100fb4` (#1908)](https://github.com/mbta/skate/pull/1908/commits/6100fb4d1d7e6a0d6cdfa616aa8ac0a6e933c37b)